ska_dbi/DBI.py

```diff
@@ -11,9 +11,12 @@
 """
 import os
 import sys
+from ska_dbi.common import DEFAULT_CONFIG, NoPasswordError
 
-supported_dbis = ('sqlite', 'sybase')
-
+if sys.version_info.major == 3 and sys.version_info.minor < 11:
+    SUPPORTED_DBIS = ('sqlite', 'sybase')
+else:
+    SUPPORTED_DBIS = ('sqlite')
```

Comment:
OK, I thought @javierggt had the requirement that we didn't want to push any changes that wouldn't work on 3.10 both so our masters tests wouldn't immediately fail and such that we'd have flexibility if we wanted to do another 3.10 release.  If that isn't a requirement this can be simpler.